lib/samples/identify_graphics/identify_graphics.dart

```diff
@@ -0,0 +1,140 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps_sdk/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class IdentifyGraphics extends StatefulWidget {
+  const IdentifyGraphics({super.key});
+
+  @override
+  State<IdentifyGraphics> createState() => _IdentifyGraphicsState();
+}
+
+class _IdentifyGraphicsState extends State<IdentifyGraphics> {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // Graphic to store the polygon.
+  late final Graphic _graphic;
+  // Graphics overlay to present the graphics for the sample.
+  final _graphicsOverlay = GraphicsOverlay();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create a map with a topographic basemap.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+    // Create a polygon geometry.
+    final polygonBuilder = PolygonBuilder.fromSpatialReference(
+      _mapViewController.spatialReference,
+    );
+    // Add points to the polygon.
+    polygonBuilder.addPointXY(x: -20e5, y: 20e5);
+    polygonBuilder.addPointXY(x: 20e5, y: 20e5);
+    polygonBuilder.addPointXY(x: 20e5, y: -20e5);
+    polygonBuilder.addPointXY(x: -20e5, y: -20e5);
+    // Create a graphic with the polygon geometry and a yellow fill symbol.
+    _graphic = Graphic(
+      geometry: polygonBuilder.toGeometry(),
+      symbol: SimpleFillSymbol(
+        style: SimpleFillSymbolStyle.solid,
+        color: Colors.yellow,
+      ),
+    );
+
+    // Add the graphics to the graphics overlay.
+    _graphicsOverlay.graphics.add(_graphic);
+    // Add the graphics overlay to the map view controller.
+    _mapViewController.graphicsOverlays.add(_graphicsOverlay);
+    // Set the viewpoint to the lake graphic.
```

Comment:
How is a "lake" involved?

Reply:
oops sorry I think when I was copying over some stuff from other sample it came along